test/test_readers.py

```diff
@@ -74,3 +74,24 @@ def test_local_recursive_nested(tmpdir):
     make(tmpdir, filenames)
     actual = LocalFileReader().list(str(tmpdir), recursive=True)
     assert parse(actual) == expected
+
```

Comment:
These should probably be in `test_images_io.py`, the `test_readers.py` are for testing generic io functionality relevant to all data types